### DISCUSSION TITLE
How to show git merge conflicts?

### DISCUSSION THREAD

**Discussion by Dou-Meishi** [JanDeDobbeleer/oh-my-posh#4223] [Troubleshoot]
I  found  this awesome theme engine before several hours and was fancinated by its flexibility and numerous beautifle theme galary. Because I previously used `bash-git-prompt` to visualize git status, I want to reproduce its effect. However, I found that there is no template variable of conflicts.  Although there is a `.Unmerged` property in `.Staging` and `.Working`, It seems that they did not show up during a merge conflict occurs.

![Screenshot from 2023-09-04 00-16-18](https://github.com/JanDeDobbeleer/oh-my-posh/assets/38120482/74da4faf-9296-45ce-a556-7ec76d1f0676)

**Comment by JanDeDobbeleer** [comment]
@Dou-Meishi can you show the output of `git status --porcelain=2` in this scenario?

**Reply by Dou-Meishi** [reply]
Sorry for the inconvenience. More information about this scenario is posted below.
![Screenshot from 2023-09-04 09-03-59](https://github.com/JanDeDobbeleer/oh-my-posh/assets/38120482/ae56731a-b7bf-4a4a-8c1a-88db5bea8be8)
